contrib/mainnet/sommelier-3/config.toml

```diff
@@ -22,4 +22,14 @@ amount = 0.000
 denom = "usomm"
 
 [metrics]
-listen_addr = "127.0.0.1:3000"
\ No newline at end of file
+listen_addr = "127.0.0.1:3000"
+
+[keys]
+delegate_key = "orchestrator" # Edit if your orchestrator's Cosmos key name is different
```

Comment:
Should we add a comment indicating `delegate_key` is only used by steward and not the orchestrator?